hw/ip/prim/dv/prim_prince/prim_prince_sim_cfg.hjson

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the name of the DUT.
+  name: prim_prince
+
+  // Top level dut name (sv module).
+  dut: prim_prince
+
+  // Top level testbench name (sv module).
+  tb: prim_prince_tb
+
+  // Simulator used to sign off this block
+  tool: vcs
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:prim_prince_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/prim/dv/prim_prince/data/prim_prince_testplan.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/dv/data/common_sim_cfg.hjson"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  overrides: [
+    {
+      name: vcs_cov_hier
+      value: "-cm_hier {proj_root}/hw/ip/prim/dv/prim_prince/data/cover.cfg"
+    }
+  ]
+
+  // List of test specifications.
+  tests: [
+    {
+      name: prim_prince_sanity
+    }
+    {
+      name: prim_prince_stress
```

Comment:
This just runs the same test a lot more times - this does not stress the DUT any more than the one above. I suggest giving it a different name. Alternately, you could remove this altogether and set the reseed for the nightly regression to 500 (do you really need 500 reruns to close the coverage?): 
```
  regressions: [ 
    {
      name: sanity
      tests: ["prim_prince_test"]
    }
    {
      name: nightly
      // Just one test to run - run it 500 times instead of the default 50. 
      reseed: 500
    }
  ] 
```

Reply:
Yes, I'm thinking to remove the testplan and just update the `sanity` and `nightly` regressions accordingly.

As per the 500 seeds for nightly regression, strictly speaking it's not at all necessary to close coverage, this large number of seeds is just to exercise more of the possible state space of key1/key2/data values, as this space is quite large.